common/src/main/java/com/microsoft/identity/common/internal/util/AndroidKeyStoreUtil.java

```diff
@@ -52,8 +53,8 @@
 import edu.umd.cs.findbugs.annotations.Nullable;
 import lombok.NonNull;
 
-import static com.microsoft.identity.common.internal.util.DateUtilities.LOCALE_CHANGE_LOCK;
-import static com.microsoft.identity.common.internal.util.DateUtilities.isLocaleCalendarNonGregorian;
+import static com.microsoft.identity.common.java.util.ported.DateUtilities.LOCALE_CHANGE_LOCK;
```

Comment:
What does the ported packagename mean?

Reply:
Utilities and functions we moved over from Common (that should probably be disposed of in the future).  This particular one appears to be just doing some examination of the calendar and date copying.  If we move to java.time, I think some of the need for such things goes away.